.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+<!--- Title -->
+
+Description
+-----------
+<!--- Describe your changes in detail -->
+
+Checklist:
+----------
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+<!--- Please refer to CONTRIBUTING.md for further guidelines -->
+- [ ] I have tested my changes. No regression in existing tests.
+- [ ] My code is Formatted.
```

Comment:
Formatted is very general term, using uncrustify ?

Reply:
Added in https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/239/commits/7dae068274c56fe8d68f7ac15ca5e1bc99ad00e0